### ISSUE TITLE
[FEATURE] Support for visionOS

### ISSUE THREAD

**Issue by svenopdehipt** [strongbox-password-safe/Strongbox#770]
Is it possible to add support for visionOS (not just the iPad app)?

**Comment by strongbox-mark** [comment]
Hi, I believe it already works on visionOS, or so we've been told...

**Comment by svenopdehipt** [comment]
Yes the iPad it is working. I asked I if would be possible to have a native app for visionOS. If you have a multiplatform app you can just add visionOS as a platform. If all dependencies are compatible, it should just compile. If you have some code which is specially marked for iOS (with `#if os(iOS)`) you can replace it with `#if os(iOS) || os(visionOS)`. I could also be that you have to change some code for visionOS (but most iOS system libs are available for visionOS). I would have tested it and created a pull request, but you mentioned it shouldn't be done and you haven't included the Xcode files so it's not so easy to test it.

**Comment by Taxyovio** [comment]
Would like to see native visionOS version as well.
